cypress/support/e2e.ts

```
@@ -0,0 +1,17 @@
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands'
```

ESLint

[error] Insert `;⏎`

(prettier/prettier)